http/src/test/resources/reference.conf

```diff
@@ -297,6 +297,12 @@ autoFreeze {
   autoFreezeCheckScheduler = 2 second
 }
 
+autoDelete {
+  enableAutoDelete= true
```

Comment:
I will have this enabled and config it to run every 1 second, otherwise, my `AutoDeleteAppMonitorSpec` will not pass. 
(Similarly, `enableAutoFreeze ` is needed for `AutopauseMonitorSpec` I think)

Reply:
Oh! But I don't see that the boolean value is actually used anywhere. Would the actual pause/delete functionality actually be turned off if you set this to `false`?

Reply:
Interesting... That config is not used which means auto pause and auto delete will always be on...
Wondering if we should remove that config if we do not have a case to disable such feature in some env?

Reply:
I think we can remove the flag...I'm also fine if u want to actually do a branching logic in `boot.scala` based on the flag

Reply:
At least I don't think we need to add the new unused config field. (it's probably a good idea to remove the one for autopause too, but that's not your problem lol)

Reply:
Adding the branching logic in Boot.scala would also be fine! And that would make it easy to disable the feature if there were some kind of issue